config/initializers/pipeline.rb

```diff
@@ -0,0 +1,19 @@
+module IdSeqPipeline
+  AWS_ACCESS_KEY_ID = ENV['AWS_ACCESS_KEY_ID']
+  AWS_SECRET_ACCESS_KEY = ENV['AWS_SECRET_ACCESS_KEY']
+  AWS_DEFAULT_REGION = ENV['AWS_DEFAULT_REGION']
+  S3_SCRIPT_LOC= 's3://czbiohub-infectious-disease/bin/pipeline.py'
```

Comment:
We should put this in this repo, so that we can version them together.

Reply:
It's in the repo. I was thinking about aws s3 cp it over during initialization

Reply:
Why cp it over at all? Like you're copying it to s3 just to copy it back.

Reply:
because. the pipeline.py is longer than 8192 characters. I couldn't submit the code directly through aegea. I need to put it on s3 and on the batch instance, copy the code to local to run.

Reply:
Oh i get it, nevermind I guess. This feels janky but we can fix it later.